bower.json

```diff
@@ -6,7 +6,7 @@
         "name": "Sudhanshu Yadav",
         "email": "sudhanshuyadav2@gmail.com"
     },
-    "version": "1.1.3",
+    "version": "2.0.0",
```

Comment:
Don't change the version here. I Will change this based on the fixes we are doing.

Reply:
done